packages/@aws-cdk/custom-resource-handlers/test/aws-s3/notifications-resource-handler/test_index.py

```diff
@@ -62,10 +62,7 @@ def make_eventbridge_configuration():
     return { EVENTBRIDGE_CONFIGURATION: {} }
```

Comment:
This PR changed this file, but the test code did not.
https://github.com/aws/aws-cdk/pull/28132